pkg/types/defaults/installconfig.go

```diff
@@ -48,11 +48,7 @@ func SetInstallConfigDefaults(c *types.InstallConfig) {
 		}
 	}
 	if c.Networking.NetworkType == "" {
-		if c.IsOKD() || c.IsSingleNodeOpenShift() {
-			c.Networking.NetworkType = string(operv1.NetworkTypeOVNKubernetes)
-		} else {
-			c.Networking.NetworkType = defaultNetworkType
-		}
+		c.Networking.NetworkType = defaultNetworkType
```

Comment:
is this even needed? with the `kubebuilder:default`, won't it always have gotten filled in if the user didn't specify it?

Reply:
is there no other way for c.Networking.NetworkType == ""

I don't know. this doesn't hurt, but if you know for certain then I can remove it.

Reply:
yeah, dunno. it doesn't hurt